README.md

```diff
@@ -64,7 +64,7 @@ $ ./startup.sh
 ### Running the test suite
 
 * Run `RAILS_ENV=test bundle exec rake db:setup` to load the database
-* Run `bundle exec rspec` to run the tests
+* Run `spring rspec` to run the tests
```

Comment:
Should this be `bundle exec spring rspec` for equivalence? Spring doesn't seem to be a globally installed gem (unless I'm missing some voodoo).

Reply:
May also want to consider an rspec binstub so could be suggested a bit more succinctly, though that may have knock-ons for other example commands I suppose.

Reply:
Sure, I've done that: my longterm hope is we can make it a system gem, [like they suggest](https://github.com/rails/spring#use-without-adding-to-bundle).

I'd like to avoid using binstubs, as at least for me, they add another layer of confusion about how to run commands. In addition, this "wrapper command form" makes it clear that it's behaving as such.